stack.yaml

```diff
@@ -3,4 +3,4 @@ packages:
 - '.'
 - 'example'
 extra-deps: []
-resolver: lts-3.1
+resolver: lts-18.18
```

Comment:
Is it bad to do this? I couldn't get it to build on my current computer with the previous resolver, but I could set it back.

Reply:
Nah -- totally reasonable. In theory, stack was supposed to be able to build any version forever, but that turned out to be more of a dream than a reality.